pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -138,6 +138,15 @@ func (proxy *DataSourceProxy) getDirector() func(req *http.Request) {
 			req.Header.Add("Authorization", util.GetBasicAuthHeader(proxy.ds.BasicAuthUser, proxy.ds.BasicAuthPassword))
 		}
 
+		if proxy.ds.secureJsonData.custumHeaders {
```

Comment:
misspelling of costum :)

Reply:
I wanted to be custom like "tailor made" but not "cast"
let's keep it ?!

Reply:
Should be `custom`